src/control/cmd/daos_agent/config.go

```diff
@@ -57,11 +57,12 @@ type Config struct {
 	FabricInterfaces    []*NUMAFabricConfig       `yaml:"fabric_ifaces,omitempty"`
 	ProviderIdx         uint                      // TODO SRS-31: Enable with multiprovider functionality
 	TelemetryPort       int                       `yaml:"telemetry_port,omitempty"`
+	TelemetryEnabled    bool                      `yaml:"telemetry_enabled,omitempty"`
```

Comment:
Is the idea of this to enable telemetry locally, with no prometheus export?

Reply:
> Is the idea of this to enable telemetry locally, with no prometheus export?

No. If it's set, telemetry is automatically enabled for all client processes. If it's not set, then clients have to enable telemetry manually using the env var. There's a config validation check that will fail if this is set to true and the port is not set.

Reply:
Ah, okay. I had thought that adding a telemetry port would be enough to indicate we wanted it enabled.

Reply:
> Ah, okay. I had thought that adding a telemetry port would be enough to indicate we wanted it enabled.

That was the original approach after we discussed it, but @mchaarawi advocated for having the option to selectively enable telemetry on a per-client basis.